pkg/trace/traceutil/otel_util.go

```diff
@@ -454,6 +454,11 @@ func GetOTelContainerTags(rattrs pcommon.Map, tagKeys []string) []string {
 	return containerTags
 }
 
+// GetOTelEnv returns the environment based on OTel resource attributes.
+func GetOTelEnv(rattrs pcommon.Map) string {
```

Comment:
nit: prefer `GetOTelEnv(res pcommon.Resource)` to avoid accidentally passing span attributes